src/__tests__/calculate.test.js

```diff
@@ -0,0 +1,101 @@
+// import { render, screen, cleanup } from '@testing-library/react';
```

Comment:
let's remove commented code. 
It’s better to delete commented code. If you need to review a previous version of your code, use git history.

I trust you can remove this before merging.

Sorry for some reason it wasn't mentioned in the previous review.